src/tls_wrap.cc

```diff
@@ -929,6 +929,16 @@ int TLSWrap::SelectSNIContextCallback(SSL* s, int* ad, void* arg) {
 #endif  // SSL_CTRL_SET_TLSEXT_SERVERNAME_CB
 
 
+void TLSWrap::GetWriteQueueSize(const FunctionCallbackInfo<Value>& args) {
+  TLSWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap, args.Holder());
+
+  wrap->UpdateWriteQueueSize();
+  args.GetReturnValue().Set(
+    wrap->object()->Get(wrap->env()->write_queue_size_string()));
```

Comment:
Can you use the overloads of `Get()` and `Set()` that take a context argument/return a `Maybe`? The others are deprecated

Reply:
Sorry @addaleax, I have very little familiarity with all of this. Do you mind elaborating? 😓

Reply:
There are two different versions of `Get()` and `Set()` each, one where the first parameter is a `context` argument and one where it is omitted (which is the deprecated one). Maybe it’s easiest to just look for the strings `Get(context` and `Set(context` in the codebase, to see how the other versions of `Get` and `Set` are used?